props/proptypes.md

```diff
@@ -101,7 +101,8 @@ export default ComponentA;
 
 ### Failing PropType Validations
 
-PropTypes give us warnings in the console whenever a component is misused.
+PropTypes give us warnings in the console whenever a component is misused. To see these warnings, or any other warnings and errors that may arise while during development, we can open our browser's developer tools (right click and select "Inspect") then select the tab called "Console". The console that is part of the developer 
```

Comment:
Students have seen and used the browser console before, so it doesn't feel to me like they need a run down on the how of seeing it. I was more concerned just that there are multiple things we refer to as the console. How about something like
```suggestion
PropTypes give us warnings in the Console tab of the browser's Developer Tools whenever a component is misused. It's important that we consult the Console tab regularly while working on our React code, since that's where the majority of React runtime errors and warnings will appear.
```

Then below, at 125:
```
When using `ComponentA`, excluding a value for `title` will produce this warning in the console:
```

Could be
```
When using `ComponentA`, excluding a value for `title` will produce this warning in the browser's Console tab:
```